.github/workflows/kind_multicluster_e2e_tests.yaml

```diff
@@ -66,6 +66,7 @@ jobs:
           - CreateMultiReaper
           - ClusterScoped/MultiDcMultiCluster
           - CreateMultiMedusa
```

Comment:
`CreateMultiMedusa` is matching both `CreateMultiMedusaOld` and `CreateMultiMedusaJob`, so both tests are being run and `CreateMultiMedusaJob` is being run twice. It should be changed to `CreateMultiMedusaOld`.

Reply:
Damn, I was pretty sure I fixed this already 🤔 
Good catch!